vizro-core/docs/pages/user-guides/custom-components.md

```diff
@@ -75,7 +75,7 @@ vm.Parameter.add_type("selector", TooltipNonCrossRangeSlider)
 ??? example "Example based on existing component"
 
     === "app.py"
-        ``` py linenums="1" hl_lines="18 19"
```

Comment:
I just checked, and it works perfectly by just adding it into the curly brackets 💪 

Example can be found at this commit: https://github.com/mckinsey/vizro/pull/569/commits/7a69eb2450747e512113bd6dd144091862c0fc08

Reply:
@maxschulz-COL @huong-li-nguyen It's not working though? It's working for highlighting, yes, but the ➕ annotations are still showing as numbers in the comments and then revealing the text at the bottom of the example.

Take a look at [the example you gave](https://vizro--569.org.readthedocs.build/en/569/pages/user-guides/custom-components/#extend-an-existing-component) and click through to the pycafe

Reply:
Yes indeed, see the comment I made here: https://github.com/mckinsey/vizro/pull/569#discussion_r1716684950

They are two different issues, one can be solved, the other one no

Reply:
Ah, gotcha! I missed that. I personally don't like the annotations that much because I don't think they're that accessible (although that's a whole other conversation with the docs, and it's unlikely to be the only issue with accessibility, right?). And that aside, it took me a while to figure out what they were so it's possible that other readers are missing them too.

Where it is possible, I am happy to work them into comments in the example, and have done this in a few places, but not consistently.

Let me go back through and (1) fix everything to be code comments where practical and (2) leave the others as annotations where there's too much (e.g. the data manager examples) because @huong-li-nguyen said she quite liked it. If we later work out a fix (or Maarten does) we still have them, and if not, it's a reasonable workaround. I think. 

Reasonable? @maxschulz-COL